lib/legacy.coffee

```diff
@@ -1,3 +1,5 @@
+fs = require('fs')
```

Comment:
module scope uppercase please.
It you uppercase this `fs` to `FS`, it won't conflict with local variable `fs`